web/admin/src/editCell.js

```diff
@@ -35,6 +35,7 @@ export class CellEdit extends Component {
         <div className="sessionCell">    
             <div className="sessionCellTop">
               <input className="sessionTitle" rows="1" name="value" maxLength="250" type="text" ref={(ip) => this.myInp = ip} onKeyPress={this.handleKeyPress} onBlur={this.handleBlur} value={this.state.value} onChange={this.handleChange} />
+              {(this.state.action !== "state") ? <p className="grayText">{250-this.state.value.length}</p> : null}
```

Comment:
Can we use the input component with max length here (and the other few locations) instead? https://doubledutch.github.io/react-components/?selectedKind=Components&selectedStory=Inputs&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel